src/packages/@ncigdc/containers/ConsequencesTable.js

```diff
@@ -77,7 +77,12 @@ const ConsequencesTableComponent = compose(
           consequence: transcript.consequence_type,
           coding_dna_change: transcript.annotation.hgvsc,
           strand: transcript.gene.gene_strand
-            ? strandIconMap[transcript.gene.gene_strand.toString(10)]
+            ? <span>
+                {strandIconMap[transcript.gene.gene_strand.toString(10)]}
+                <ForTsvExport>
```

Comment:
I guess I mean more if your are going to do more work to support the temporary solution, why not do the easy solution that can be permanent :) But I won't block this PR about it or anything, so it's up to you.